web/modestio/settings.py

```diff
@@ -124,7 +124,8 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.1/howto/static-files/
 
-STATIC_URL = "static/"
+STATIC_URL = "/static/"
+STATIC_ROOT = os.path.join(BASE_DIR, "static/")
```

Comment:
`BASE_DIR` is a `pathlib.Path` object, so you can use the syntax sugar it offers: `path / "subpath"` instead of the traditional `os.path.join(path, "subpath")`.